swagger.yaml

```diff
@@ -4021,6 +4020,25 @@ definitions:
       type:
         $ref: '#/definitions/image_type'
 
+  host_static_network_config:
+    type: object
+    properties:
+      network_yaml:
+        type: string
+        description: yaml string that can be processed by nmstate
+      mac_interface_map:
+        $ref: '#/definitions/mac_interface_map'
+
+  mac_interface_map:
+    type: array
+    items:
+      type: object
+      properties:
+        mac_address:
+          type: string
+        nic_name:
```

Comment:
would it be more precise to call it 'nic_alias' ? since it is a temp name used to refer a nic which its name is unknown at the time of providing the parameters

Reply:
changed to logical_nic_name